schema/spec.md

```diff
@@ -348,6 +357,25 @@ Versions for a schema named `com.example.telemetrydata`:
 }
 ```
 
+#### `validation`
+
+- Type: Boolean
+- Description: Indicates whether or not the server will validate the Resource's
+  document(s) against the specified `format` attribute. A value of `true`
+  indicates that the server MUST reject any request that would cause any
+  Version of this Resource to be invalid per the rules as defined by the
+  `format` specification. Note, this includes a request to set this attribute
```

Comment:
I don't understand the note - is it saying "If there's a request to set this attribute to true, the server must validate the rest of the intended state of the resource document immediately"? (So you don't get to set it to true with an invalid document.)

Reply:
Yes, and in particular it means all Versions of the Resource need to be checked too since it seems like a lie to me to have "validation"=true AND have some Versions of the Resource be non-compliant per the "format" attribute. Would it help if I elaborated more  in the spec?

Reply:
Definitely worth elaborating more. I think it's also worth *considering* the use case of "I had broken resources before, but anything from now on should be valid" (in terms of any *new* versions). Or is the intention that if you had broken ones and you want validation, you should delete all the broken ones first?

Reply:
yes, I was assuming that people would be forced to update all existing Versions before they could turn on validation. Annoying, but I think that's better than allowing for things to be in an inconsistent state.  That's why I have a bit of a reservation about allowing "old ones" to be non-compliant... how do we explain to new users of that Resource why some are invalid or even what the date is for the switch? "Required compliance" seems like the easier path. But let's discuss on the call.

Added a bit more text to make this clear.